curriculum/challenges/english/25-front-end-development/quiz-javascript-functions/66edcc779993c0da6906dbb9.md

```diff
@@ -17,439 +17,439 @@ Answer all of the questions below correctly to pass the quiz.
 
-Placeholder answer
+`myFunction = () => { ... };`
 
 ### --question--
 
 #### --text--
 
-Placeholder question
+What is the difference between a regular function and an arrow function?
 
 #### --distractors--
 
-Placeholder distractor 1
+Arrow functions do not have their own 'this' binding.
 
 ---
 
-Placeholder distractor 2
+Arrow functions cannot be used as constructors.
 
 ---
 
-Placeholder distractor 3
+Arrow functions cannot be used with the 'new' keyword.
 
 #### --answer--
 
-Placeholder answer
+All of the above.
 
 ### --question--
 
 #### --text--
 
-Placeholder question
+You can also call arrow function as 
 
 #### --distractors--
 
-Placeholder distractor 1
+Lambda function
 
 ---
 
-Placeholder distractor 2
+Fat arrow function
 
 ---
 
-Placeholder distractor 3
+Neither A or B
 
 #### --answer--
 
-Placeholder answer
+Both A and B
 
 ### --question--
 
 #### --text--
 
-Placeholder question
+What is the benefit of using arrow functions?
 
 #### --distractors--
 
-Placeholder distractor 1
+You can omit curly braces and return keyword if your output is going to be a single line.
 
 ---
 
-Placeholder distractor 2
+They provide easy syntax while writing promises and callbacks.
 
 ---
 
-Placeholder distractor 3
+They have short syntax and increase readability.
 
 #### --answer--
 
-Placeholder answer
+All of the above.
 
 ### --question--
 
 #### --text--
 
-Placeholder question
+What is the difference between a function and a method?
 
 #### --distractors--
 
-Placeholder distractor 1
+There is no difference between functions and methods.
 
 ---
 
-Placeholder distractor 2
+Functions and methods are properties of objects
 
 ---
 
-Placeholder distractor 3
+Functions are properties of objects, while methods are standalone functions.
 
 #### --answer--
 
-Placeholder answer
+Methods are properties of objects, while functions are standalone functions.
 
 ### --question--
 
 #### --text--
 
-Placeholder question
+Can a method be used outside of its object?
 
 #### --distractors--
 
-Placeholder distractor 1
+Yes, methods can be used anywhere a function can be used.
 
 ---
 
-Placeholder distractor 2
+Methods can be used outside of their object, but only if they are bound to a specific 'this' value.
 
 ---
 
-Placeholder distractor 3
+Methods can be used outside of their object, but only if they are defined as arrow functions.
 
 #### --answer--
 
-Placeholder answer
+No, methods can only be used within their object.
 
 ### --question--
 
 #### --text--
 
-Placeholder question
+Which of the following is an example of a method?
 
 #### --distractors--
 
-Placeholder distractor 1
+`function greet(name) { console.log("Hello, " + name); }`
 
 ---
 
-Placeholder distractor 2
+`var greet = function(name) { console.log("Hello, " + name); };`
 
 ---
 
-Placeholder distractor 3
+All of the above.
 
 #### --answer--
 
-Placeholder answer
+`const person = { name: "Alice", greet: function() { console.log("Hello, my name is " + this.name); } };`
```

Comment:
I think it would look better multiline

````suggestion
```js
const person = {
    name: "Alice",
    greet: function() {
        console.log("Hello, my name is " + this.name);
    }
};
```
````

Reply:
Updated.